pkg/apply/apply.go

```diff
@@ -106,11 +106,10 @@ func ApplyObject(ctx context.Context, client cnoclient.Client, obj Object, subco
 		}
 	}
 
-	fieldManager := "cluster-network-operator"
+	fieldManager := GetFieldManager(subcontroller)
```

Comment:
wait a second, I don't think we want to change fieldManager here, because it will cause the same problems when migrating from one fieldManager to another.
I think we wanted to leave Apply field manager as it was (as it is explicitly specificed), and only update field manager used by the client?

Reply:
@npinaeva this change is made to have uniform prefix on the field manger value , it also does force apply on the object. so where do you see a problem ?

Reply:
fields that are owned now by "cluster-network-operator/subcontroller" will be renamed to "network-operator/subcontroller" which requires one more field manager deprecation and we didn't want that?
cc @kyrtapz

Reply:
Yeah, I don't think we need to unify the prefix in this change. We can do that separately in the future if required.

Reply:
Ok @npinaeva / @kyrtapz , as discussed, retaining code to use "cluster-network-operator/subcontroller".